src/test/java/teammates/storage/sql/CoursePostgresDbTest.java

```diff
@@ -0,0 +1,47 @@
+package teammates.storage.sql;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import teammates.test.BaseTestCaseWithLocalPostgresSqlDatabaseAccess;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import javax.sql.DataSource;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * SUT: {@link CoursesDb}.
+ */
+public class CoursePostgresDbTest extends BaseTestCaseWithLocalPostgresSqlDatabaseAccess {
+    private final CoursesDb coursesDb = CoursesDb.inst();
+    private static PostgreSQLContainer pgsql;
+
+    @BeforeClass
+    public static void startContainer() {
+        pgsql = new PostgreSQLContainer<>("postgres:14.4")
+                .withDatabaseName("pgsql-test")
+                .withUsername("pgsql-test")
+                .withPassword("pgsql-test");
```

Comment:
Do you think it is possible to move this to the `BaseTestCaseWithLocalDatabaseAccess` class, so that we don't need to repeat this code for all the test classes requiring local database access?

Reply:
These few lines of code are just for testing that the test container works. They will be removed when real tests are written.

Reply:
Yes. For now I will move it to `BaseTestCaseWithLocalPostgresSqlDatabaseAccess` before the `BaseTestCaseWithLocalDatabaseAccess ` is eventually removed.